spec.md

```diff
@@ -5,17 +5,20 @@ Introduction
 ------------
 
 This document aims to specify an ewasm VM in a way useful to contract writers and VM implementers.
-To this end, multiple things are specified:
-
--   The extra state that a VM needs to have around to successfully respond to calls into the EEI.
--   The EEI (Ethereum Environment Interface), currently specified loosely [here](eth_interface.md).
 
 ### Terminology
 
 **EEI**: The *Ethereum Environment Interface* refers to the layer between the Ethereum Client code and the execution engine.
-         **TODO**: Is *EEI* the right term to use?
 
-**Ethereum Client**: Code which can interact with the blockchain (read/validate and sending transactions).
+**EEI method**: A method exposed by the Ethereum Client to the execution engine for interacting with the blockchain state.
```

Comment:
I wander maybe "function" would be more familiar name.

Reply:
I wanted to avoid the word "function" because they aren't functions in the mathematical sense. I think "method" or "procedure" is closer, in the sense of methods from OOP languages like Java which can alter the state of the surrounding context.